data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3483 @@
+			choice
+				`	(Go to the stadium.)`
+				`	(Ignore them and go back to my ship.)`
+					defer
+			`	As you head towards the temple, moving with the crowd, you can start to hear words being spoken loudly. A couple with 4 children is speaking, or more precisely, begging, "... and we fear the next season will be even more rough. We need foundings to restore the lands, nothing grows on them anymore!" The "Chiefs" speak together, and then grant her her request, before the next one in line comes. The sun is almost set, and you can see that the Chiefs are growing a bit impatient. "My cousins would like to join me, but they lack the founds to afford it. Would you allow more to join us, and become true?" This seems to bother them, they answer briefly, "We will contact True ship captains to ask them if they can, but I fear they are indeed busy these days, which is why the prices have gone up." Then they rise up, and one speaks to the crowd, "It is late, you would do well to go home to care for those near you. Never hesitate to come back to us in the next cycle should you need help." At this, the crowd leaves, as do the Chiefs later on. You notice one of them looking at you, pensive, as he just noticed your presence.`
+			`	The same Chief is now moving in your general direction. He seems to be lost in his thought and is mumbling on, something about, "We do not get alien visitors often here. There has to be a reason for your presence..." When he speaks again, directly to you, it is with more focus, "Sorry, I was just thinking out loud! Well you are here anyway so maybe you could help us?"`
+			`	"We have friends outside of our space, civilians and others. You see, we would do anything to help a friend, and a few Hai want to join us. I can only hope you are willing to help us reunite with our brothers and sisters from Hai space," he says, apparently a bit desperate, considering he knows very little if anything at all about you.`
+			choice
+				`	"Of course, what do you need me to do?"`
+					goto action
+				`	"What do you mean, brother and sisters from Hai space? Are you not different?"`
+					goto brethren
+				`	"Why do you need me to do it?"`
+				`	"This seems like a bad idea, ask someone else."`
+					decline
+			
+			label whyme
+			`	"Well normally they would be able to pay to come here, but the price is pretty high; too high. The Hai make it difficult for us to go anywhere; as you have probably noticed they have a fleet at the edge of our system. And not many Hai captains are willing to bring them here, or at least they ask for a big fee, because of the obvious risks involved, like getting killed in all that cross fire. Yet you somehow made your way here, probably also under fire from our own people, but now that we are allied, you hopefully should not have any issues getting out of here. So, are you willing to help us?"`
```

Comment:
```suggestion
			`	"Well, normally they would be able to pay to come here, but the price is pretty high; too high. The Fettered Hai make it difficult for us to go anywhere; as you have probably noticed they have a fleet at the edge of our system. And not many Hai captains are willing to bring them here. If they do they ask for a big fee because of the obvious risks involved, like getting killed in all that cross fire. Yet you somehow made your way here, probably also under fire from our own people, but now that we are allied, you hopefully should not have any issues getting out of here. So, are you willing to help us?"`
```